bundles/org.openhab.binding.sunsynk/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sunsynk"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<!-- Bridge Thing Type -->
+
+	<bridge-type id="sunsynkaccount">
+		<label>SunSynk Account</label>
+		<description>Access to SunSynk Account. Used to discover plants and inverters tied to account.</description>
+		<properties>
+			<property name="vendor">SunSynk</property>
+		</properties>
+		<config-description>
+			<parameter name="email" type="text" required="true">
+				<label>E-mail Address</label>
+				<description>E-mail address for your SunSynk Connect account.</description>
+				<context>email</context>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for your SunSynk Connect account.</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="access_token" type="text" required="false">
+				<label>SunSynk Secret</label>
+				<description>Secret for accessing SunSynk Connect services.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refresh_token" type="text" required="false">
+				<label>SunSynk Refresh Secret</label>
+				<description>SunSynk Connect services refresh secret.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="expires_in" type="integer" required="false">
+				<label>SunSynk Secret expiry</label>
+				<description>SunSynk Connect services token expiry in seconds.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="issued_at" type="integer" required="false">
+				<label>SunSynk Secret issue time stamp</label>
+				<description>SunSynk Connect services token issue time stamp.</description>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+
+	</bridge-type>
+
+
+	<!-- Inverter Thing Type -->
+	<thing-type id="inverter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="sunsynkaccount"/>
```

Comment:
Minor, but the binding id allready makes it unique, no need to add sunsynk again.
```suggestion
			<bridge-type-ref id="account"/>
```

Reply:
Done